pywbemtools/pywbemcli/_cmd_connection.py

```diff
@@ -125,27 +133,40 @@ def connection_select(context, name):
     connection. If the NAME argument is omitted, prompt for selecting one of
     the connection definitions in the connections file.
 
-    This command is useful in the interactive mode of pywbemcli.
+    Selects a connection from the persistently stored set of named connections
+    if NAME exists in the store. If NAME not supplied, a list of connections
+    from the connections definition file is presented with a prompt for the
+    user to select a connection.
 
-    The selection of the current connection is not saved across interactive
-    sessions of pywbemcli.
+    Selection is persistent; it is used as the server definition in commands
+    in the remainder of an interactive session and is saved in the
+    connections file and used as the current connection if there is no other
+    server definition (--server or --name or --mock-server).
 
     Examples:
 
-      pywbemcli
+      $ pywbemcli
 
-      > connection select myconn
+      pywbemcli> connection select myconn
+      pywbemcli> connection show
+      pywbemcli> CTRL-D
+      $ pywbemcli show
+        TODO
+        ...
```

Comment:
This does not format right. You need to put each line between blank lines. Given that, this is not the place to explain larger sequences of commands. I suggest to remove the show commands.

Reply:
DONE used \b to simplify